diviner/analysis/pmdarima_analyzer.py

```diff
@@ -184,15 +182,14 @@ def calculate_is_constant(self):
         """
         self._create_group_df()
         group_constant_check = {
-            group: is_constant(group_df[self._y_col])
-            for group, group_df in self._group_df
+            group: is_constant(group_df[self._y_col]) for group, group_df in self._group_df
         }
         return group_constant_check
 
     @experimental
     def calculate_acf(
         self,
-        unbiased=None,
```

Comment:
Is `False` a more appropriate default than `None` here? I'm fine with the change if so, just want to make sure :)

Reply:
In 2.x they did some refactoring from the previous `None` default to a `False` (which makes more sense based on the consumer of that arg). Just standardizing here to something that looks less confusing. 
Ref on 2.x version (with the arg renaming handler code): https://github.com/alkaline-ml/pmdarima/blob/b465ee3dfa1df4819c4be9bb3e62e531873a779b/pmdarima/utils/wrapped.py#L60-L98

Reply:
Sounds good :D